app/controllers/users_controller.rb

```diff
@@ -0,0 +1,20 @@
+class UsersController < ApplicationController
```

Comment:
Ruby standard code guideline use 2 tabs rather than 4 tab senpai ^^

So, we can improve the code with install Rubocop, and make that tools to fix the indentation issues in this repo
https://github.com/rubocop/rubocop